yt/yt/server/query_tracker/query_tracker_proxy.cpp

```diff
@@ -46,6 +46,8 @@ static TLogger Logger("QueryTrackerProxy");
 
 namespace NDetail {
 
+constexpr TStringBuf EveryoneShareAccessControlObject = "everyone-share";
```

Comment:
Can we add a comment above?

`// Hidden from ListQueries, but allows access by query id to [everyone | people who have access to this ACO].`

or

`// This ACO is the same as all others, but does not affect ListQueries. This is needed to enable link sharing.`